packagehandlers/packagehandlers_test.go

```diff
@@ -344,6 +344,43 @@ func TestUpdateDependency(t *testing.T) {
 				descriptorsToCheck: []string{"package.json"},
 			},
 		},
+
+		// Conan test cases
+		{
+			{
+				vulnDetails: &utils.VulnerabilityDetails{
+					IsDirectDependency:          true,
+					SuggestedFixedVersion:       "3.0.14",
+					VulnerabilityOrViolationRow: formats.VulnerabilityOrViolationRow{Technology: techutils.Conan, ImpactedDependencyDetails: formats.ImpactedDependencyDetails{ImpactedDependencyName: "openssl", ImpactedDependencyVersion: "3.0.9"}},
+				},
+				scanDetails:        scanDetails,
+				fixSupported:       true,
+				testDirName:        filepath.Join("conan", "conan_txt"),
```

Comment:
rename conan dir to "conan" instead "conan_txt" and you can remove testDirName since it will take the name of the dir according to the tested tech